lib/blob/blobstore.c

```diff
@@ -5789,6 +5789,15 @@ spdk_blob_calc_used_clusters(struct spdk_blob *blob)
 	return num;
 }
 
+void
+spdk_blob_reset_used_clusters_cache(struct spdk_blob *blob)
+{
+	assert(blob != NULL);
+	if (spdk_blob_is_thin_provisioned(blob)) {
+		blob->num_used_clusters_cache = 0;
```

Comment:
I think we need to take the lock before updating it, at least current code is doing that.

Reply:
Thanks for finding. I will fix it.